src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Tree/Dependencies/GraphNodes/Actions/TrackChangesGraphActionHandler.cs

```diff
@@ -73,18 +75,21 @@ public override bool HandleChanges(IGraphContext graphContext, SnapshotChangedEv
 
                 using (var scope = new GraphTransactionScope())
                 {
-                    viewProvider.TrackChanges(
+                    if (viewProvider.TrackChanges(
                         graphContext,
                         projectPath,
                         updatedDependency,
                         inputGraphNode,
-                        updatedSnapshot.Targets[updatedDependency.TargetFramework]);
+                        updatedSnapshot.Targets[updatedDependency.TargetFramework]))
+                    {
+                        anyChanges = true;
+                    }
 
                     scope.Complete();
                 }
             }
 
-            return false;
+            return anyChanges;
```

Comment:
Was this a bug that it always returned `false`?

Reply:
The return value wasn't being used before.